nn_modules/decoder/decoder_test.py

```diff
@@ -0,0 +1,29 @@
+'''
+Author: diwei diwei@qcraft.ai
+Date: 2024-06-15 13:40:53
+LastEditors: diwei diwei@qcraft.ai
+LastEditTime: 2024-06-15 17:19:14
+FilePath: /top_eleven/nn_modules/decoder/decoder_test.py
+Description: 
+
+Copyright (c) 2024 by ${git_name_email}, All Rights Reserved. 
+'''
+
+
+import unittest
+import torch
+from decoder import *
```

Comment:
I have the impression that import * is not a recommended practice.

Reply:
Make sense.Done.